functions/src/endpoints/experiments.endpoints.ts

```diff
@@ -58,6 +65,52 @@ export const createExperiment = onCall(async (request) => {
 
       // Create the stages
       for (const stage of data.stages) {
+        // If payout stage, use payout config to generate scoring config
+        if (stage.kind === StageKind.Payout) {
+          const getScoringQuestion = (question: RatingQuestionConfig) => {
```

Comment:
This is great! Just a flag that we may want to generalize this moving forward (with different question types, such as multiple choice).